settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
That was probably working on previous AS versions, but if we don't include `analyzer` explicitly, it doesn't get  picked up as a module by Android Studio.